sys-libs/libatomic_ops/libatomic_ops-7.10.0.recipe

```diff
@@ -20,8 +20,8 @@ CHECKSUM_SHA256="0db3ebff755db170f65e74a64ec4511812e9ee3185c232eeffeacd274190dfb
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
-libVersion="1.4.0"
-libGplVersion="1.4.0"
+libVersion="1.3.0"
+libGplVersion="1.3.0"
```

Comment:
@ivmai I'm not sure if there is a change both libraries differ in verion, if not this could be reduced to only:
```
libVersion="1.3.0"
```

Reply:
There are changes in both `libatomic_ops.a` and `libatomic_ops_gpl.a` since v7.8.2. Full source diff is: https://github.com/bdwgc/libatomic_ops/compare/v7.8.2...v7.10.0

Reply:
Given that there are separate variables for that in the source, I would also keep them separate in the recipe. They could be different again in later versions.

Reply:
@ivmai those variables are defined as you mention, static libraries are not packaged when shared ones are, so the information in the *.a ones are invalid for us.
The SONAME is computed during build, so something is setup wrong there?